### PR TITLE
Development branch

### PR DESCRIPTION
**Pull request by J03lMATH**

### CODE REVIEW

**Comment by J03lMATH** [comment]
Finished the prior Checkstyles

File: src/ComputeEngineAPI.java [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
+    public int compute(int input1, int input2) {
+        return ackermann(input1, input2);
     }
-    //Delegating jobs to the Client, ComputeEngine, and the DataStorage
-    public ComputingResult compute(ComputingRequest request) {
-        ComputingResult comRequest=new ComputingResult();
-        return comRequest;
+
+    // Computes the result using the ackermann function
+    public int ackermann(int m, int n) {
```
Comment (sbrandon96) [inline src/ComputeEngineAPI.java]: Very clean ackermann!

**Comment by J03lMATH** [comment]
Before you do more work just approve and comment 
and do more work on a different branch

**Review by Youssup — APPROVED** [review]
Fixed ty

**Pull request merged**